### ISSUE TITLE
[4.0.0-beta.11] Popover position incorrect in RTL mode

### ISSUE THREAD

**Issue by abennouna** [ionic-team/ionic#15662]
# Bug Report

**Ionic Info**
```
Ionic:

   ionic (Ionic CLI)          : 4.1.2 (~/.nvm/versions/node/v8.11.1/lib/node_modules/ionic)
   Ionic Framework            : @ionic/angular 4.0.0-beta.11
   @angular-devkit/core       : 0.7.5
   @angular-devkit/schematics : 0.7.5
   @angular/cli               : 6.2.2
   @ionic/ng-toolkit          : 1.0.8
   @ionic/schematics-angular  : 1.0.6

System:

   NodeJS : v8.11.1 (~/.nvm/versions/node/v8.11.1/bin/node)
   npm    : 5.6.0
   OS     : macOS High Sierra
```

**Describe the Bug**
When a document is in RTL mode, the popover starts on the absolute left relative to the clicked element.

**Steps to Reproduce**
1. Use a document with `dir="rtl"` and `md` mode
2. Add a popover component
3. After presenting, if error not obvious, inspect and lower popover container opacity

**Related Code**
I can upload a code if necessary.

**Expected Behavior**
The right position of the popover should be aligned to the right of the clicked element.

**Additional Context**
- Tested only in browser in `md` mode.
- In `ionic/core/src/components/popover/animations/md.enter.ts`, we can change this 
```
  const targetLeft =
    targetDim != null && 'left' in targetDim
      ? targetDim.left
      : bodyWidth / 2 - contentWidth / 2;
```

to
```
    const direction = document.documentElement.getAttribute('dir');
    const targetLeft =
      targetDim != null && 'left' in targetDim
        ? direction === 'rtl'
          ? targetDim.left - contentWidth + targetDim.width
          : targetDim.left
        : bodyWidth / 2 - contentWidth / 2;
```

**Comment by abennouna** [comment]
Animation in RTL mode should also be inverted.

**Comment by abennouna** [comment]
Closed in favor of #17012
